tools/wake-format/README.md

```diff
@@ -0,0 +1,481 @@
+# wake-format
+
+`wake-format` is a tool for automatically applying the wake style guide to wake source files. It is intentionally opinionated to create a singular correct formatting standard.
+
+# Usage
+
+The standard usage of `wake-format` is to format a set of wake files in place.
+
+`wake-format -i source.wake source2.wake source3.wake`
+
+`wake-format` supports other flows for various use cases. The `-n`/`--dry-run` flag is ideal for CI/CD to check if the source needs to be formatted. It exits with a non-zero status code when a file would be changed by `wake-format`
+
+Running `wake-format` without `-i` will emit the file to `stdout` instead of overwritting it.
+
+# Documentation
+
+The information below should be helpful to contribute to `wake-format`
+
+## Building
+
+`wake-format` is built in the same manner as `wake`. Running `make` or `./bin/wake build default` from within the wake dir will produce the `wake-format` binary.
```

Comment:
> or `./bin/wake build default` from within

This implies that they've already run `make` at least once, it might be nice to add a Makefile target for the formatter even if it just calls wake under the hood

Reply:
https://github.com/sifive/wake/pull/975 should fix that, I'll update the documentation once it lands

Reply:
I think users that would prefer `make wake-format` can just call `make` instead.